test/selection.js

```diff
@@ -1,6 +1,5 @@
 /* eslint-disable consistent-return, no-undef, no-unused-expressions */
 import chai, { expect } from "chai";
-import { spy } from "sinon";
 import sinonChai from "sinon-chai";
 import { env } from "jsdom";
```

Comment:
Actually sinon isn't used in the whole test file, so we remove sinon-chai too and limit the imported stuff to `import { expect } from "chai";`.